scripts/config/create-config.ts

```diff
@@ -0,0 +1,221 @@
+      DapiServer: operationsRepository.chains[chainName].contracts.DapiServer || '',
+    };
+    const options: ChainOptions = {
+      fulfillmentGasLimit: 500000,
+      gasPriceOracle: [
+        {
+          gasPriceStrategy: 'constantGasPrice',
+          gasPrice: {
+            value: 10,
+            unit: 'gwei',
+          },
+        } as const,
+      ],
+    };
+    const providers = {
+      [`provider_${sanitiseFilename(chainName).replace(/\-/g, '_')}`]: {
+        url: `\${${sanitiseFilename(chainName).replace(/\-/g, '_')}_PROVIDER_URL}`.toUpperCase(),
+      },
+    };
+
+    return {
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm' as const,
+      options,
+    };
+  });
+
+  const nodeSettings = {
+    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
+    airnodeAddress: airnode,
+    airnodeXpub: xpub,
+    logFormat: 'plain',
+    logLevel: 'INFO',
+  } as NodeSettings;
+
+  const apiCredentials = Object.values(ConfigurationData.ois).flatMap((ois: any) =>
+    Object.keys(ois.apiSpecifications.components.securitySchemes).map((security) => ({
+      oisTitle: ois.title,
+      securitySchemeName: security,
+      securitySchemeValue: `\${SS_${sanitiseFilename(security).toUpperCase()}}`.replace(/ /g, '_').replace(/\-/g, '_'),
+    }))
+  );
+
+  const oisSecrets = Object.values(ConfigurationData.ois).flatMap((ois: any) =>
+    Object.keys(ois.apiSpecifications.components.securitySchemes).map((security) =>
+      `SS_${sanitiseFilename(security).toUpperCase()}=`.replace(/ /g, '_').replace(/\-/g, '_')
+    )
+  );
+
+  const airkeeperSecretsArray = [
+    'AIRNODE_WALLET_MNEMONIC=""',
+    ...apiChains.map((chainName) => `${sanitiseFilename(chainName).replace(/\-/g, '_')}_PROVIDER_URL=`.toUpperCase()),
+    ...oisSecrets,
+  ];
+
+  const awsSecretsArray = [
+    `AWS_ACCESS_KEY_ID=`,
+    `AWS_SECRET_ACCESS_KEY=`,
+    `AWS_SESSION_TOKEN=`,
+    `REGION=us-east-1`,
+    `STAGE=dev`,
+  ];
+
+  const airkeeperSubscriptions: Subscriptions = Object.values(ConfigurationData.beacons)
+    .flatMap((beacon: any) =>
+      Object.entries(beacon.chains).map(([chainName, chain]: any) => {
+        const chainId = parseInt(operationsRepository.chains[chainName].id);
+        const dapiServerInteface = getDapiServerInterface();
+        const parameters = '0x';
+        const airnodeAddress = airnode;
+        const templateParameters = encode(beacon.template.decodedParameters);
+        const endpointId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['string', 'string'],
+            [beacon.template.oisTitle, beacon.template.endpointName]
+          )
+        );
+        const templateId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [endpointId, templateParameters]);
+        const threshold = ethers.BigNumber.from(100000000)
+          .mul(chain.updateConditionPercentage * 100)
+          .div(10000);
```

Comment:
Do we really need to multiply by 100 and then divide by 10000? Can't we just divide by 100?

```suggestion
        const threshold = ethers.BigNumber.from(100_000_000)
          .mul(chain.updateConditionPercentage)
          .div(100);
```

Reply:
yeah because if the value is something like 0.01 then it doesn't work as expected